### PR TITLE
For Elixir Language Image #1171

### PR DESCRIPTION
**Pull request by c0b**
docker-library/official-images#1171
docker-library/official-images#1398

### CODE REVIEW

**Comment by c0b** [comment]
CI failing on short-description too long (>100 characters); so I cut it a little bit; please comment here if it doesn't describe very well

1. from http://elixir-lang.org/ it is `Elixir is a dynamic, functional language designed for building scalable and maintainable applications.` (102 chars);
2. from [wikipedia](https://en.wikipedia.org/wiki/Elixir_%28programming_language%29) it is `Elixir is a functional, concurrent, general-purpose programming language that runs on the Erlang virtual machine (BEAM).` (120 chars)

File: elixir/README-short.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Elixir is a dynamic, functional language for building scalable and maintainable applications.
```
Comment (c0b) [inline elixir/README-short.txt]: short description has to be within 100 chars.

Comment (yosifkit) [inline elixir/README-short.txt]: Yeah, that is a docker hub restriction :disappointed:.

**Comment by yosifkit** [comment]
LGTM

**Comment by tianon** [comment]
LGTM

**Pull request merged**